receive.go

```diff
@@ -190,7 +191,7 @@ func (r *receiver) run(ctx context.Context) error {
 					r.mu.Unlock()
 				}
 				i++
-				cp := &currentPath{path: p.Stat.Path, stat: p.Stat}
+				cp := &currentPath{path: filepath.FromSlash(p.Stat.Path), stat: p.Stat}
```

Comment:
This does not look correct. The protobuf types on the wire should already always be unix normalized. These kind on conversions should be on the walker when the file is the actual local reference. In here it is remote reference with no local mapping.